app/views/complaints/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="details-header">
+  <div class="grid-container">
+    <h1>Issue #<%= @complaint[:id] %></h1>
+  </div>
+</div>
+<div class="details-body grid-container">
+  <div class="grid-row grid-gap">
+    <div class="grid-col-4">
+      <div class="summary-section">
+        <h2>Summary</h2>
+        <p>
+          <strong><%= @complaint.dig :attributes, :grantee %>:</strong>
```

Comment:
[suggestion] I think, unless if there's a good reason not to do this, that I would recommend parentheses for `.dig` since it makes it a bit easier to read.  At the same time, it got past the linter so that at least doesn't think it's a big deal?

Reply:
➕